src/test/java/io/pakland/mdas/githubstats/domain/service/GetTeamsFromOrganizationTest.java

```diff
@@ -0,0 +1,26 @@
+package io.pakland.mdas.githubstats.domain.service;
+
+import io.pakland.mdas.githubstats.domain.model.Organization;
+import io.pakland.mdas.githubstats.domain.model.Team;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class GetTeamsFromOrganizationTest {
+    @Test
+    public void givenOrganization_shouldReturnListOfTeams() {
+        Team t = new Team();
```

Comment:
This is a preference, but I would name this variable `team` instead to increase readability. It also looks more consistent to me with how you named the list `teams` (singular/plural for object/list)